### ISSUE TITLE
Error "extended entry already exists"

### ISSUE THREAD

**Issue by CoffeeFanatic81** [rootCBR/DisruptManager#5]
Hello,

first of all - thanks for all the hard work you do on Disrupt Engine tools.

I'm trying to use Disrupt Manager to modify localization files of Watch Dogs: Legion and it works great, but when I try to modify /ui/fonts/orwellbold_default.ffd (or any other .ffd in /ui/fonts/) I do get an exception "extended entry already exists" - is there anything I am doing wrong or is it a bug in the Disrupt Manager? Is there some other way to modify fonts on Watch Dogs: Legion using your tools?

Thanks a lot,
CF

**Comment by rootCBR** [comment]
What you're doing should work in theory, even though I'm not sure how one would modify a .ffd format file. I can take a closer look if you send me the files you are trying to pack.

**Comment by CoffeeFanatic81** [comment]
I'm using FFDConverter for the fonts, but it basically happens when I try to add any file to the /ui/fonts/ - it even happens when I try to use the original font assets and try to repack them using your Disrupt Manager. I prepared an example patch for your tools here https://mega.nz/file/WjQQ1BCY#aRR1xgqZbaDW5KNGhOhD9nImV-K_bSINbi1K5ns-7K4 - the exception happens with this file for example. I appreciate a lot your help, thanks!

**Comment by rootCBR** [comment]
I've created a new release that should fix this problem.

**Comment by CoffeeFanatic81** [comment]
Thanks a lot @rootCBR it's working perfectly and I was able to successfully modify the font to include my language special letters. Great job!

**Comment by CoffeeFanatic81** [comment]
Sorry to bothering you again, I just noticed one thing with the new version - it can not update the /languages/*.loc files. I didn't noticed first since I had the languages updated from your previous version of Disrupt Manager, but the current version won't handle the /languages/ now (previous version did).

I verified that it is not caused by the fonts (I used clean .fat and .dat files) and I tried language update that worked with previous version and it seems to succeed, but it is not updated within the game.

I also noticed (hopefully it helps you) that when I update the language using previous Disrupt Manager version, then I delete the .bak file and I update the /ui/fonts/ using latest Disrupt Manager version - it works and I have both language and font there. It's just that it's not possible to do it using single patch and latest version. Thanks for looking into this.
